genesis/utils/mjcf.py

```diff
@@ -209,6 +209,7 @@ def parse_link(mj, i_l, scale):
     l_info["invweight"] /= scale**3
     for j_info in j_infos:
         j_info["pos"] *= scale
+    j_infos = [j_info for j_info in j_infos if j_info["n_dofs"] > 0]
```

Comment:
Add comment to explain what you are doing and why this is important.

Reply:
Mujoco removes all fixed joints ? Ok but do you get their pose ? This may be convenient for the user. What about rather filtering the fixed joints in the unit tests instead ?

Could you check if this is true for the head of the stickman ?

Reply:
yeah, mujoco does not fixed joint.